subscriber/podaac_data_downloader.py

```diff
@@ -244,7 +284,7 @@ def run(args=None):
                       results['items']]
     downloads_metadata = [[u['URL'] for u in r['umm']['RelatedUrls'] if u['Type'] == "EXTENDED METADATA"] for r in
                           results['items']]
-    checksums = pa.extract_checksums(results)
+    checksums = pa.extract_checksums(results['items'])
```

Comment:
I'd use `results.get('items', [])` to either return an empty list if somehow `items` key doesn't exist; but if that check is happening before / guaranteed to be returned then it works!

Reply:
Good point Randy -- luckily CMR always returns `items` even when the query matches 0 results

Reply:
Roger that; consider this resolved!